src/get.rs

```diff
@@ -129,14 +132,99 @@ impl AsyncRead for DataStream {
     }
 }
 
+/// Gets a collection and all its blobs using a [`Ticket`].
+pub async fn run_ticket<A, B, C, FutA, FutB, FutC>(
+    ticket: &Ticket,
+    keylog: bool,
```

Comment:
Heh, true doc comments for these run functions have some way to go.  I don't really like the functions in the first place, they would be better as builders instead of having all these options.  But an earlier attempt at doing that failed as there was some weirdness I forgot.

Anyway: the keylog boolean enables the rustls [KeyLog](https://docs.rs/rustls/latest/rustls/struct.KeyLogFile.html) functionality.  If you set it to `true` and run the application with `SSLKEYLOGFILE` environment variable set to a filename the TLS pre-master key will be written into that file and can be used by e.g. wireshark to inspect the captured QUIC traffic.